cf-suse-stacks.yml

```diff
@@ -63,23 +73,39 @@ jobs: ##########################################################################
     - get: version
       params: { pre: alpha }
     - get: receipt
+
   - do:
     - task: make-rootfs
       privileged: true
       file: src-ci/tasks/make-rootfs/task.yml
       input_mapping:
         src: stacks-git
+
   - put: artifact
     params:
       file: out/opensuse42-*.tar.gz
       acl: public-read
+
   - put: version
     params: { file: version/version }
+
   - put: receipt
     params:
       file: out/opensuse42-*.receipt.txt
       acl: public-read
+
   - put: receipt-diff
     params:
       file: out/opensuse42-*.receipt.diff
       acl: public-read
+
+  - put: docker-image
+    params:
+      import_file: out/opensuse42.tar.gz
+      skip_download: true
```

Comment:
> "put" has an implied "get"

Ok, that makes sense.

Digging in the docs: https://concourse.ci/put-step.html

> Additionally, you can control the settings of the implicit get step by setting get_params. For example, if you did not want a put step utilizing the docker-image resource type to download the image, you would implement your put step as such:

```
plan:
- put: docker-build
  params: {build: git-resource}
  get_params: {skip_download: true}
```